docs/ossm/ossm2-migration/gateway-migration.md

```diff
@@ -0,0 +1,92 @@
+# Gateway Migration Guide
+
+Migrating gateways between Istio control planes during a version upgrade from 2.6 to 3.0 is very similar to migrating regular workloads, here is some information on how to migrate your gateways.
+
+## Migration Scenarios
+
+### Gateway Canary Migration (Recommended)
+
+For gradual rollout using multiple gateway versions (avoiding any downtime):
```

Comment:
(avoiding any downtime) implies that the in place upgrade _will_ result in downtime which it shouldn't. Maybe we can just say that this option is for users who want more control of their gateway rollout similar to the upstream language.

Reply:
Fair, made this change here: [7bbca39](https://github.com/openshift-service-mesh/sail-operator/pull/217/commits/7bbca39a365f05d52fa16257827819db6c34214c)